File: Soal4.py
def cek_duplikat(angka):
    return len(angka) != len(set(angka))

while True:
    try:
        inputan = input("Masukkan angka: ")
        
        angka = list(map(int, inputan.split()))
        hasil = cek_duplikat(angka)
        if hasil == True:
            print(f"Hasil: {hasil}")
            print("Ada angka yang duplikat")
            break
        else:
            print(f"Hasil: {hasil}")
            print("Tidak ada angka yang duplikat")
            break
    except ValueError:
        print("Input tidak valid. Masukkan angka yang benar.")

File: Soal4.js
function cek_duplikat(angka){
    return angka.length !== new Set(angka).size
}

input = [20, 1, 3, 2, 4, 6, 8, 5, 7, 9, 11, 13, 15, 10,  12, 14, 16, 18, 20, 17, 19] 

if (cek_duplikat(input) == true) {
    console.log("Input: " + input)
    console.log("Output: " + cek_duplikat(input))
    console.log("\nAda angka yang duplikat")
}else {
    console.log("Input: " + input)
    console.log("Output: " + cek_duplikat(input))
    console.log("\nTidak ada angka yang duplikat")
}